test/performance/odp_pktio_ordered.c

```diff
@@ -188,13 +188,13 @@ typedef union ODP_ALIGNED_CACHE {
  * IPv4 5-tuple
  */
 typedef struct {
-	int32_t src_ip;
-	int32_t dst_ip;
-	int16_t src_port;
-	int16_t dst_port;
-	int8_t  proto;
-	int8_t  pad0;
-	int16_t pad1;
+	uint32_t src_ip;
```

Comment:
Fixing the types in the ipv4_tuple fixes also potential problems (implementation defined value or signal raised) in calc_flow_idx(). Maybe make some remark in the commit message?